tests/framework/ROM/TimeSeries/SyntheticHistory/TrainingData/AutoCorrelation_A.csv

```diff
@@ -0,0 +1,2 @@
+scaling,filename
+1,AutoCorrelation_A_0.csv
```

Comment:
I noticed this file is pretty long. If that's what it takes to effectively test this, and it's still a fast test, no problem. Otherwise, can we get by with 100 entries instead?